src/Neo.CLI/CLI/MainService.cs

```diff
@@ -392,25 +391,32 @@ public async void Start(CommandLineOptions options)
                     }
                     else
                     {
-                        DisplayError("DLL not found, please get libleveldb.dll.");
+                        DisplayError("DLL not found, please get libleveldb.dll.",
+                            "Download from https://github.com/neo-ngd/leveldb/releases");
                     }
                 }
-                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                else if (OperatingSystem.IsLinux())
+                {
+                    DisplayError("Shared library libleveldb.so not found, please get libleveldb.so.",
+                        "Use command \"sudo apt-get install libleveldb-dev\" in terminal or download from https://github.com/neo-ngd/leveldb/releases");
+                }
+                else if (OperatingSystem.IsMacOS() || OperatingSystem.IsMacCatalyst())
                 {
                     DisplayError("Shared library libleveldb.dylib not found, please get libleveldb.dylib.",
-                        "From https://github.com/neo-project/neo/releases");
+                        "Use command \"brew install leveldb\" in terminal and download from https://github.com/neo-ngd/leveldb/releases");
                 }
                 else
                 {
                     DisplayError("Neo CLI is broken, please reinstall it.",
-                        "From https://github.com/neo-project/neo/releases");
+                        "Download from https://github.com/neo-project/neo/releases");
                 }
-
+                return;
```

Comment:
Or you kill the process or it's better to remove this return

Reply:
`LevelDbStore` is the problem here. So best to `kill`?

Reply:
with this return you don't process the node completly.

Reply:
But the way it sits now. If `NeoSystem` throws an exception then `NeoSystem.AddService` and anything else needing `NeoSystem` will crash the process anyways. So best to return here and close the process gracefully.

Reply:
I didn't tested, is closed the process?

Reply:
`DisplayError` already `exits` process. I'm just fixing warning and error messages
![image](https://github.com/neo-project/neo/assets/8141309/888b8bab-f9ac-4e1f-b5b4-6a6dece81547)